dnit/docker.ts

```diff
@@ -0,0 +1,65 @@
+// Utils for running (adl or terraform) in docker:
+
+import {currentUserOpts, dockerRun, dockerRunConsole} from './deps.ts'
+
+/** Run dockerized terraform image - interactive IO */
+export async function runDockerizedTerraform(cmds: string[]): Promise<void> {
```

Comment:
Would like to see this replaced with a adl style cached platform specific download (but not for this PR).